.gitignore

```diff
@@ -1,2 +1,2 @@
-.vscode/
-node_modules
```

Comment:
(This is a test comment for me to have a feel of how PRs are done)

node_modules indeed need not be included when committing it in the repository. I learned somewhere that as long as you the package.json, you should not worry about the dependencies.

Reply:
Happy learning!